### DISCUSSION TITLE
Welcome to ptprashanttripathi Discussions!

### DISCUSSION THREAD

**Discussion by PtPrashantTripathi** [PtPrashantTripathi/ptprashanttripathi#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Manish-Nirala** [comment]
You are doing very good to create Sanskrit programming language

**Reply by PtPrashantTripathi** [reply]
It's just a small step to bring Vedic Indian languages in modern world programming 🙏

**Reply by chirag127** [reply]
How to use that Sanskrit programming language?

**Reply by PtPrashantTripathi** [reply]
@chirag127 please visit the tutorial page of VEDIC programming language https://vedic-lang.github.io

**Comment by Chavda-Mitul** [comment]
hey would love to connect with you

**Reply by PtPrashantTripathi** [reply]
Hi mail me at ptprashanttripathi@outlook.com

**Comment by Sarkar069** [comment]
Thanks so much for making the Bhagavad Gita API available

**Comment by Idiot-12998** [comment]
Bro , I want to make my own programming language in sanskrit from scratch can you tell me what do i need to do in order to make my own .

**Comment by Utshow327** [comment]
Do you know a way to get the hinduisms main books to json or have links of
all json files?

I am new to github the link i clicked took me to archive.org

I am making this app for all religions and is being difficult for me to
find them

Do you know a way?

**Reply by PtPrashantTripathi** [reply]
Hi Utshow,

I wish there was a magic “Convert Sacred Texts to JSON” button somewhere, but sadly, no such thing exists. The truth is, you’ll have to roll up your sleeves and go through the grind regex gymnastics, some Python scripting, and plenty of patience to parse the PDFs/texts/Markdowns and shape them into clean JSON.

Think of it like tapasya (penance) for developers — the reward comes only after you go through the ritual of cleaning and structuring the data yourself 🙃.

So, short answer: you’ll need to do it yourself.

Best of luck with your project — it sounds like a wonderful initiative!

Regards,
Prashant

**Comment by syerrawa** [comment]
Hey @PtPrashantTripathi 

Great work in setting this up. I can imagine the amount of work involved here. 
Qq: Could you point me to the source for translations by various authors?

Thanks in advance.

**Reply by PtPrashantTripathi** [reply]
Source books: https://github.com/vedicscriptures/bhagavad-gita-api#-source-book

**Comment by Utshow327** [comment]
Wth is happening why emails appearing in my email lol, i just my vibecoder
to get all the best mit licensed hinduism data 😆

**Reply by PtPrashantTripathi** [reply]
@Utshow327 Since you previously commented on this thread, GitHub is likely sending you email updates. You can easily stop these by opening one of the notification emails and clicking the Unsubscribe link at the bottom, or by clicking the Unsubscribe button on the right sidebar of this page.
